samcli/commands/package/command.py

```diff
@@ -81,6 +82,7 @@ def resources_and_properties_help_string():
 @check_newer_version
 @track_template_warnings([CodeDeployWarning.__name__, CodeDeployConditionWarning.__name__])
 @print_cmdline_args
+@unsupported_command_cdk(alternative_command=None)
```

Comment:
Should this be running `cdk deploy`? CDK's deploy also does the package.

Reply:
I wasn't 100% sure on this, but if you think it's the correct alternative then I'll add that in.